### DISCUSSION TITLE
Getting error in import.

### DISCUSSION THREAD

**Discussion by Rahul251952** [Cyfrin/foundry-full-course-cu#2986] [General]
![Screenshot (1)](https://github.com/user-attachments/assets/c8cb03d4-dfcd-41f8-8b18-9426856115b0)
getting error in import and the error is:
Source "forge-std/Test.sol" not found

**Comment by EngrPips** [comment]
Hello @Rahul251952, run `forge install foundry-rs/forge-std --no-commit` in your terminal that should install the library.

**Reply by Rahul251952** [reply]
Thanks man.

**Reply by EngrPips** [reply]
Anytime, Friend. Keep crushing it.

**Comment by CoheeYang** [comment]
this is mostly because the vscode cannot read it, but forge can read the path, try to add a remapping.txt with the following commands:

> forge remappings > remappings.txt

**Reply by EngrPips** [reply]
Interesting.
